plugins/discovery-gce/src/test/resources/org/elasticsearch/discovery/gce/compute/v1/projects/metadataserver/zones/europe-west1-b/instances

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I think that we should replace those JSON files by simple code in the test class (not in this PR). I'm not even sure that they are all used.

Reply:
they are used in GceDiscoveryTests#testMetadataServerValues

https://github.com/elastic/elasticsearch/pull/33721/files#diff-9a2669df7c9d3ea8a5a5100ee4f78951R274